src/components/ui/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import type { BadgeProps } from '@faststore/ui'
+
+import { Badge } from '.'
+
+export default {
+  component: Badge,
+  title: 'Components/Badge',
+}
+
+const Template = ({ children, ...args }: BadgeProps) => (
+  <Badge {...args}>{children}</Badge>
+)
+
+export const Default = Template.bind({})
+
+Default.args = {
+  children: 'Something',
+  small: false,
+  interactive: false,
+  variant: 'highlighted',
```

Comment:
So the example looks like the discount badge we have on the PDP:
```suggestion
  children: '50% off',
  small: true,
  interactive: false,
  variant: 'neutral	',
```